modules/metering-install-operator.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-operator_{context}"] = Installing the Metering Operator
+
+To install the Metering Operator first create a dedicated OpenShift project for
+Metering, and then install the Metering Operator.
+
+.Procedure
+
+. In the {product-title} web console, click *Project* -> *Create Project*.
+
+. Select a title for the project (we recommend metering), and click *Create
+Project*.
+
+. Next, click *Catalog* -> * OperatorHub*, and search for "metering" to find the
+Metering Operator.
+
+. Click the Metering card to open its package description, then click Install.
+
+. In the Create Operator Subscription screen, select the metering namespace you
+created above in the *A specific namespace on the cluster* drop-down, and
+specify your update channel and approval strategy. Click *Subscribe* to install
+the metering-operator into your selected namespace.
+
+. On the *Subscription Overview* screen, the *Upgrade status* indicates 1
```

Comment:
This is a personal preference, but I would reword this to be a bit clearer about the `1 installed`. What about:

> ...the *Upgrade status* indicates when the Metering Operator has finished installing. Once installed, this count will be `1`.